docs/docs/reference/release-notes/image-migration-guide.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Migrating from gatsby-image to gatsby-plugin-image
+---
+
+This document is provided to help users of gatsby-image migrate to using gatsby-plugin-image.
+
+Note that you can use both packages at the same time and may need to in situations where you're using a gatsby-image compatible CMS as a data source and that CMS plugin has not yet updated to a version compatible with the new API.
+
+If you're looking for other documentation on the new plugin, see:
+
+- How to guide: //todo
+- Reference guide: //todo
+
+## What changed?
+
+### New Syntax
+
+The new plugin requires significant syntax changes. We've provided a [codemod](#codemod-instructions) to help you migrate, but this section will explain the changes.
+
+#### Import change
+
+Previously, `GatsbyImage` was the default export from `gatsby-image`. This is no longer the case.
```

Comment:
Wording is a little confusing. GatsbyImage is still the default export from `gatsby-image`. Might be better to say something thing like "...but the new component is a named export from "gatsby-plugin-image""